### PR TITLE
Java 8 v371

### PR DESCRIPTION
**Pull request by JariP10**
Java 8 v371 changes

### CODE REVIEW

**Review by haavistoj — APPROVED** [review]
looks ok

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <groupId>org.robotframework</groupId>
     <artifactId>javafxlibrary</artifactId>
     <packaging>jar</packaging>
-    <version>0.7.1-SNAPSHOT</version>
+    <version>0.7.1</version>
```
Comment (sampeson) [inline pom.xml]: Development branch should be SNAPSHOT, this could be 0.7.1-SNAPSHOT until changed to new version.

File: pom.xml [hunk 2]
```diff
@@ -164,207 +170,207 @@
...
+                    </archive>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.robotframework</groupId>
+                <artifactId>robotframework-maven-plugin</artifactId>
+                <version>1.7.1</version>
```
Comment (sampeson) [inline pom.xml]: there is new version 2.1.0, check this and all other dependencies and should them be updated

File: pom.xml [hunk 3]
```diff
@@ -105,31 +111,31 @@
                     </plugin>
                     <!-- Upload also documentation -->
                     <plugin>
-						<groupId>org.codehaus.mojo</groupId>
```
Comment (sampeson) [inline pom.xml:108]: Why these lines has changed?

File: pom.xml (hunk 1 above)
Comment (JariP10) [reply pom.xml]: Fixed

File: pom.xml (hunk 2 above)
Comment (JariP10) [reply pom.xml]: Fixed necessary dependencies

**Review by ataura — APPROVED** [review]
Näyttää hyvältä

File: pom.xml [hunk 4]
```diff
@@ -58,6 +58,12 @@
             <email>pasi.saikkonen@eficode.com</email>
             <organization>Eficode</organization>
             <organizationUrl>http://www.eficode.com</organizationUrl>
+        </developer>
+		 <developer>
```
Comment (sampeson) [inline pom.xml]: Intend totally different?

File: src/.gitignore [hunk 5]
```diff
@@ -1,3 +0,0 @@
-target
```
Comment (sampeson) [inline src/.gitignore:1]: Why these are removed, I beliewve we don't want to put target directory to version control

**Review by sampeson — APPROVED** [review]

File: src/.gitignore (hunk 5 above)
Comment (JariP10) [reply src/.gitignore:1]: This file is not needed.

File: pom.xml (hunk 4 above)
Comment (JariP10) [reply pom.xml]: Intend fixed.

File: pom.xml (hunk 3 above)
Comment (JariP10) [reply pom.xml:108]: Intend fixed in merge master.

**Review by ataura — COMMENTED** [review]
Ei huomautettavaa

**Pull request merged**